scripts/build

```diff
@@ -21,10 +20,24 @@ var template = fs.readFileSync(path.join(__dirname, 'template.js'),
 //  version :: String
 var version = require('../package.json').version;
 
-//  $pathEq = [String] -> a -> Object -> ()
-var $pathEq = R.curry(function(path, expected, obj) {
-  assert.strictEqual(R.path(path, obj), expected);
-});
+//  logWithPrefix = String -> String -> ()
+var logWithPrefix = R.curryN(2, R.pipe(
+  R.concat,
+  R.concat(_, "\n"),
+  function(str) {
+    process.stderr.write(str);
+  }
+));
+
+//  warn :: String -> ()
+var warn = logWithPrefix("\x1b[33m>>> WARNING: \x1b[39;49m");
```

Comment:
I recall reading that `\uXXXX` should be preferred over `\xXX`, though I can't find the source. I'd prefer to use `\u001B` here regardless.